docs/Introduction_to_Backend_Architecture.md

```diff
@@ -0,0 +1,123 @@
+# KubeFATE 部署FATE支持后端介绍
+
+FATE支持的计算引擎eggroll和spark
+
+## 支持后端
+
+FATE支持选择不同的计算引擎，存储引擎和数据传输引擎，现在KubeFATE支持快捷的部署不同引擎组合的FATE集群。
+
+目前支持的组合包括以下几种：
+
+- Eggroll
+- Spark + hdfs + RabbitMQ
+- Spark + hdfs + Pulsar
+- Spark-local + localfs + Pulsar
+
+### Eggroll
+
+Eggroll是FATE支持计算存储引擎，包括rollsite，nodemanager，clustermanager三个组件
+
+rollsite负责数据传输，nodemanager负责存储和计算，clustermanager负责管理。
+
+docker-compose使用的时候修改`parties.conf`配置
+
+```bash
+backend=eggroll
+```
+
+k8s使用的时候修改`cluster.yaml`配置
+
+```yaml
+backend: eggroll
+```
+
+架构图：
+
+<div align="center">
+  <img src="./images/arch_eggroll.png" />
+</div>
+
+### spark_rabbitmq
+
```

Comment:
We need some buffering between chapters, like:
What is the drawback of eggroll and why we need rabbitmq.